src/docs/howto_develop.md

```diff
@@ -181,6 +182,13 @@ not your fork. If you are posting a review request, put the pull request number
 field. Then, when you close the request, you can navigate from the bug number to easily close
 the pull request.
 
+If your CI-build failed in Travis-CI, and the failure looks like it's not due to
+your change, please open an issue with the part of the CI log containing the test failure and label
+the issue with `flaky-test`. If an issue already exists, add a comment to it noting that you
+encountered it too. After you've done that, you can ask in chat for someone to restart the shard.
```

Comment:
`chat` could be ambiguous for folks who don't realize we have Slack. Could reword to `you can ask in Slack`.